docs/journalist.rst

```diff
@@ -594,7 +594,7 @@ Decrypting and Preparing to Publish
 .. note::
 
    To decrypt a VeraCrypt drive on a Windows or Mac workstation, you need
-   to have the VeraCrypt software installed. If you are unsure if you have the
+   to have the **VeraCrypt** software installed. If you are unsure if you have the
```

Comment:
The bolding here seems a bit excessive, I think unless we are explicitly directing folks to a menu or part of the UI as part of referencing the name of the application, we can omit it (applicable to other additions of bolding for software names as well)

Reply:
That makes sense, I'll review the other instances too.

Reply:
Done.